third_party/Windows-CalcEngine/cmake/WCECompilerFlags.cmake

```diff
@@ -18,8 +18,8 @@ IF ( CMAKE_COMPILER_IS_GNUCXX OR "x${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang" )
         ADD_CXX_DEFINITIONS("-fPIC")
     endif()
     ADD_CXX_DEFINITIONS("-pedantic") # Turn on warnings about constructs/situations that may be non-portable or outside of the standard
-    ADD_CXX_DEFINITIONS("-ffor-scope")
-    ADD_CXX_DEFINITIONS("-Wall -Wextra") # Turn on warnings
+    #ADD_CXX_DEFINITIONS("-ffor-scope")
+    #ADD_CXX_DEFINITIONS("-Wall -Wextra") # Turn on warnings
```

Comment:
@vidanovic I just muted some warnings in our build of WCE.  No functional change.